iosApp/iosApp.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -176,17 +176,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/mapbox/turf-swift.git",
       "state" : {
-        "revision" : "1997fb3c3dc0d8181690a1ec192ed907bcead9e8",
-        "version" : "3.0.0"
+        "revision" : "bf840e6b9529d105687840fe2c9dcd74197d46d1",
+        "version" : "4.0.0"
```

Comment:
I got a bunch of linker errors after this turf upgrade. Clearing DerivedData, cleaning the build folder, and re-building resolved.